addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19638,7 +19644,11 @@ msgctxt "#36203"
 msgid "Enables the \"Personal Video Recorder\" (PVR) features. This requires that at least one PVR add-on is installed."
 msgstr ""
 
-#empty string with id 36204
+#. Description of setting with label #22078 "Default play action"
+#: system/settings/settings.xml
+msgctxt "#36204"
+msgid "Toggle between [Play] (default) and [Resume].[CR][Play] will automatically play videos from the beginning or if a resume point is present ask whether to play from beginning or to resume.[CR][Resume] will automatically resume videos from the last position that you were viewing them. Play automatically from beginning otherwise."
```

Comment:
I runtime tested this as a regular user without looking into the implementation and I found this a bit confusing (same as @the-black-eagle according to his comments). The "Play" action is actually a variant of "Choose" which will prompt the user (if a resume point is available) to play from the beginning or resume. Otherwise it will just play.
Can we somehow do it better with the action name?
I know this has very little to do with the actual implementation and comes down to user experience. Unfortunately I also don't have many ideas here. This is explained in the help text but the string is quite big and takes a lot of time to scroll - not very intuitive for a regular user.

Reply:
> not very intuitive for a regular user

I do agree. If you have some idea for better wording, please. Just "Choose" will not work, for two reasons:

1) What will happen is not always a choice (if no resume point is set)
2) "Choose" is taken for video select action already, popping up a menu with more choices than just resume/play (also "information" and "queue" and "more" which will open the full popup menu. Here the wording fits because there will always be choices.

Reply:
Points above also apply to "play"..
What if we name the setting around resume instead of play? Something around the lines of:

Default playback resume action:
- Ask
- Always resume 

It has the benefit of being pretty clear it only applies for resume points. Implicitly "play" falls out of the bounds of such checks and...just plays.

Reply:
"Always resume": Will users understand that start from beginning, which will happen if no resume point set, is also kind of resume?

Reply:
Well, personally I think that if no resume point is set the "Default playback resume action" doesn't really apply. The action taken is always play regardless of "ask" or "always resume"

Reply:
We have plemty of space on the settings page:

"Ask if resumable, play from beginning otherwise"
"Resume if possible, play from beginning otherwise"

What about this?

Reply:
![screenshot00005](https://github.com/xbmc/xbmc/assets/3226626/18ac91b9-4aef-4a99-8b27-55d3e58d7ebe)
![screenshot00006](https://github.com/xbmc/xbmc/assets/3226626/73c976ce-bbc9-4b9a-becf-a688870b935b)

Reply:
@enen92 so, something like this (note that I also renamed the setting label itself):

![screenshot00008](https://github.com/xbmc/xbmc/assets/3226626/a1fe1f7a-cfc7-4ef7-bd59-b2555d3dc1af)
![screenshot00009](https://github.com/xbmc/xbmc/assets/3226626/a9109818-d1ba-4305-8bb0-02e891cf95e0)

Only problem with that would be if we in the future want to add other values, which are not related to resume. Maybe somebody wants to bring up the info dialog when you press play on your remote. Then, would be back to where w started with the naming discussion. Just saying.

Reply:
your suggestion looks good to me and makes sense if for some reason it needs to be extended in the future (unlike mine).
Anyway, can always be changed in the future so maybe go with your suggestion for now

Reply:
> maybe go with your suggestion for now

Which one exactly?

"Default playback resume action" for setting label
"Ask" + "Always resume" for the values?

Reply:
No the "verbose" one linked to the "play" action instead of mine (which binds to resume action)

Reply:
> No the "verbose" one linked to the "play" action instead of mine (which binds to resume action)

Okay.

Reply:
I also do not like those long strings... let's for now just go with:

msgid "Default play action"
msgid "Ask if resumable"
msgid "Resume"

... and explain in the help text the fallback to play from beginning.

Reply:
> I also do not like those long strings... let's for now just go with:
> 
> msgid "Default play action" msgid "Ask if resumable" msgid "Resume"
> 
> ... and explain in the help text the fallback to play from beginning.

Missed this on first read, yes I could live with too